src/org/sosy_lab/java_smt/test/InterpolatingProverTest.java

```diff
@@ -987,6 +987,43 @@ public <T> void bigSeqInterpolationTest() throws InterruptedException, SolverExc
     }
   }
 
+  @Test
+  @Ignore
+  public <T> void testTrivialInterpolation() throws InterruptedException, SolverException {
```

Comment:
What is the goal of this test?
You are testing the same behavior twice here.

Reply:
I fixed the test to validate discussed behavior. 
https://github.com/sosy-lab/java-smt/pull/320#discussion_r1307348386